File: NSEC.Music_Player/Media/NotificationButtonReceiver.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Support.V4.Media.Session;
using Android.Views;
using Android.Widget;
using Newtone.Core.Logic;

namespace NSEC.Music_Player.Media
{
    [BroadcastReceiver(Enabled = true, Exported = false)]
    [IntentFilter(new[] { "android.intent.action.MEDIA_BUTTON" })]
    public class NotificationButtonReceiver:MediaButtonReceiver
    {
        public override void OnReceive(Context context, Intent intent)
        {
            KeyEvent ev = (KeyEvent)intent.GetParcelableExtra(Intent.ExtraKeyEvent);

            if(ev.Action == KeyEventActions.Down)
            {
                if (ev.KeyCode == Keycode.MediaPlay)
                {
                    MediaPlayerHelper.Play();
                }
                else if (ev.KeyCode == Keycode.MediaPause)
                {
                    MediaPlayerHelper.Pause();
                }
                else if (ev.KeyCode == Keycode.MediaPrevious)
                {
                    MediaPlayerHelper.Prev();
                }
                else if (ev.KeyCode == Keycode.MediaNext)
                {
                    MediaPlayerHelper.Next();
                }
            }
        }
    }
}

File: app/src/main/java/com/nejman/nsec/music_player/media/NotificationButtonReceiver.java
package com.nejman.nsec.music_player.media;

import android.content.Context;
import android.content.Intent;
import android.view.KeyEvent;

import androidx.media.session.MediaButtonReceiver;

public class NotificationButtonReceiver extends MediaButtonReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        if(intent == null)
        {
            return;
        }

        KeyEvent ev = (KeyEvent) intent.getParcelableExtra(Intent.EXTRA_KEY_EVENT);

        if(ev == null)
        {
            return;
        }

        if(ev.getAction() == KeyEvent.ACTION_DOWN)
        {
            System.out.println("down");
            if(ev.getKeyCode() == KeyEvent.KEYCODE_MEDIA_PLAY)
            {
                MediaPlayerHelper.play();
            }
            else if(ev.getKeyCode() == KeyEvent.KEYCODE_MEDIA_PAUSE)
            {
                System.out.println("ev.getKeyCode() == KeyEvent.KEYCODE_MEDIA_PAUSE");
                MediaPlayerHelper.pause();
            }
            else if(ev.getKeyCode() == KeyEvent.KEYCODE_MEDIA_PREVIOUS)
            {
                MediaPlayerHelper.prev();
            }
            else if(ev.getKeyCode() == KeyEvent.KEYCODE_MEDIA_NEXT)
            {
                MediaPlayerHelper.next();
            }
        }
    }
}
